css/main.css

```diff
@@ -191,3 +191,9 @@ h3 {
     font-family: Georgia, 'Times New Roman', Times, serif;
 }
 
+/*accordion for menu page */
+
+.accordion-collapse ul {  
+    max-height: 250px;
+    overflow-y: scroll;
```

Comment:
Ok, I get what the _attempt_ was but this doesn't actually fix the issue. The accordion element should stick at the top of the page while everything inside of the accordion should be allowed to scroll under and past it.

Reply:
![image](https://user-images.githubusercontent.com/888231/111567983-4cd74880-8776-11eb-8016-6d64aff49b18.png)

Reply:
ok, will rework!